packages/common/src/styles/useAppTheme.ts

```diff
@@ -1,11 +1,17 @@
 import { Direction } from '@mui/material/styles';
 import { Theme } from '@mui/material';
 import { useState, useEffect } from 'react';
-import theme from './theme';
+import { themeOptions, createTheme } from './theme';
 import { useRtl } from '@common/intl';
+import { useLocalStorage } from '../localStorage';
+import merge from 'lodash/merge';
 
 export const useAppTheme = (): Theme => {
-  const [currentTheme, setTheme] = useState(theme);
+  const [customTheme] = useLocalStorage('/theme/custom');
+  const appTheme = customTheme
+    ? createTheme(merge(themeOptions, customTheme))
```

Comment:
merge allows for a partial of theme options to be stored, which makes overriding easier and safer